src/components/simulator/ContextExplorer.tsx

```diff
@@ -131,10 +125,6 @@ export default class ContextExplorer extends React.Component<
     }
 
     const newPath = [...path, name];
-    if (EXCLUDED_PATHS[newPath.join('.')]) {
```

Comment:
this isn't needed anymore - `child.run` and `parent.run` aren't in the autocomplete files